examples/3d_scene.rs

```diff
@@ -20,10 +20,22 @@ fn setup(
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
-    let camera_entity = Entity::new();
     // add entities to the world
     commands
-        // plane
+        // camera
+        .spawn(Camera3dComponents {
+            transform: Transform::new(Mat4::face_toward(
+                Vec3::new(-3.0, 5.0, 8.0),
+                Vec3::new(0.0, 0.0, 0.0),
+                Vec3::new(0.0, 1.0, 0.0),
+            )),
+            ..Default::default()
+        });
+
+    let camera_entity = commands.current_entity().unwrap();
```

Comment:
I think you can simplify this block by changing it to
```rust
let camera_entity = commands
    .spawn(nonsense here)
    .current_entity().unwrap();
```
This will also reduce the chance of someone (me) adding another spawn block before the `camera_entity` definition, which would make it point to some entity other than the camera.

Reply:
Updated